galaxy/values.yaml

```diff
@@ -18,6 +18,1765 @@ webHandlers:
 jobHandlers:
   replicaCount: 1
 
+galaxyConf:
+  - name: jobConf
+    mountPath: /galaxy/config/job_conf.xml
+    rawContents: |
+      <job_conf>
+          <plugins>
+              <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner" workers="4"/>
+          </plugins>
+          <handlers assign_with="db-skip-locked" />
+          <destinations>
+              <destination id="local" runner="local"/>
```

Comment:
What is our strategy for secret values? For example, as I recall, job_conf can have API keys etc. So should it perhaps be in a secrets map instead?